### PR TITLE
feat: changes title to Web Identity (WebID) 1.0

### PR DESCRIPTION
**Pull request by jacoscaz**
This PR changes the title of the identity spec as per #54 .

As this change would have no practical consequence, deadline for review is set to 2024-04-08.

Links to the rendered files:

- main branch: https://w3c.github.io/WebID/spec/identity/index.html
- this PR: https://htmlpreview.github.io/?https://github.com/jacoscaz/WebID/blob/ci/feat/title-web-identity/spec/identity/index.html

### CODE REVIEW

**Comment by melvincarvalho** [comment]
Noticing the removal of names like Henry Story and Tim Berners-Lee, replaced by yours with "**version=1.0**," prompts concern. If I may, with all due humility, inquire: do all original contributors consent to these changes? Creating a new specification is within your purview, yet this approach may appear as misattribution. Deeply regrettable, if so. I must express strong reservation.

**Comment by jacoscaz** [comment]
_/chair hat on_

@melvincarvalho while I don't particularly appreciate the subtext in your comment, you do make a good point. There's no trace of Henry in the working draft as it stands right now and that's something that needs to be rectified right away. I'm going to list him as a past chair (was he also an editor?) but two questions for you:

- Do you think it would be a good idea and would you be willing to work on a PR that extends the "acknowledgements" section with a few lines dedicated to Henry? I could also do this but, regrettably, I did now know him outside of the very limited exchanges I had with him in the last few years and I feel it would be better if this were crafted by someone who did know him. Some of you have known him for decades, I think, and actually met him in person over the years.
- Could you point out where TimBL's name has been removed from?

**Comment by melvincarvalho** [comment]
I think the main problem here is that of versioning.  WebID 1.0 ED IMHO should belong to Henry, TimBL and Andrei forever.

A fork with a higher version number would make it clear it's new work.  Trying to keep 1.0 and change the names and text, is problematic, IMHO.

**Comment by melvincarvalho** [comment]
> * Do you think it would be a good idea and would you be willing to work on a PR that extends the "acknowledgements" section with a few lines dedicated to Henry?

Yes, that could work, but I would much rather do it under a version `1.1` which would keep a clear delineation between the version worked on by Henry, Timbl and Andrei, and the new 2024 version.

This would also mirror the approach of JSON-LD 1.0 and 1.1 where JSON-LD 1.1 was an incremental improvement on 1.0 but 1.0 remained intact.

File: spec/identity/index.bs [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 <pre class="metadata">
 
-Title: Web Identity and Discovery
+Title: Web Identity (WebID) 1.0
```
Comment (melvincarvalho) [inline spec/identity/index.bs:6]: I'd much rather this had a higher version, such as 1.1

That would elegantly resolve the issues between the historical spec from 2014, and the new work in 2024

Everyone gets properly attributed, and no sweeping changes are needed.

**Comment by melvincarvalho** [comment]
Perhaps it's a good idea to wait until we get a new chair/editor, before merging the current set of PRs.

File: spec/identity/index.bs (hunk 1 above)
Comment (pmcb55) [reply spec/identity/index.bs:6]: Why not just remove the version number completely (as this is `Title:` metadata, and versioning metadata (if any) should be completely separate), i.e., :
```suggestion
Title: Web Identity (WebID)
```
?

**Comment by TallTed** [comment]
> [@melvincarvalho] the approach of JSON-LD 1.0 and 1.1

and
> I'd much rather this had a higher version, such as 1.1

JSON-LD is a substantially different case, not least because JSON-LD 1.0 is a full REC (TR), having passed through PR after CR emerged from ED.

WebID has _still_ not yet reached CR, never mind the later stages, and the WebID 2014 ED should not ever be referred to as if it had, which calling the 2014 ED "WebID 1.0" or calling the current work "WebID 1.1" would effectively do.

**Pull request closed without merging**